sdk/documentintelligence/Azure.AI.DocumentIntelligence/tests/DocumentIntelligenceClient/DocumentBatchLiveTests.cs

```diff
@@ -18,7 +18,7 @@ public DocumentBatchLiveTests(bool isAsync)
         {
         }
 
-        [Test]
+        [RecordedTest]
```

Comment:
Missed this in the last PR. We're supposed to use `RecordedTest` to enable all Test Framework features.